scc/provider/resource_subaccount.go

```diff
@@ -325,11 +336,23 @@ func (r *SubaccountResource) Update(ctx context.Context, req resource.UpdateRequ
 		return
 	}
 
-	// if shouldUpdateTunnel(plan) {
-	// 	if err := r.updateTunnelState(ctx, plan, state, endpoint, &respObj, &resp.Diagnostics); err != nil {
-	// 		return
-	// 	}
-	// }
+	if !plan.Connected.IsNull() && !plan.Connected.IsUnknown() {
```

Comment:
The sync of the connection should be added to the CREATE function too when considering my comment from above.